SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -137,6 +140,12 @@ USpatialNetDriver::USpatialNetDriver(const FObjectInitializer& ObjectInitializer
 #endif
 
 	SpatialDebuggerReady = NewObject<USpatialBasicAwaiter>();
+
+	if (GetDefault<UGeneralProjectSettings>()->bSpatialAuthorityDebugger)
```

Comment:
Whilst this code (and code elsewhere) is behind a 'feature toggle', should be also remove it completely from shipping and/or testing builds?